domains/bennguyen.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "benvgaming",
+        "email": "benvipgaming@protonmail.com"
+    },
+    "record": {
+        "CNAME": "https://ben-homepage.vercel.app/"
```

```suggestion
        "CNAME": "ben-homepage.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`). Perhaps you meant to use a URL record?